flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamTaskTest.java

```diff
@@ -410,6 +410,42 @@ public CloseException() {
         }
     }
 
+    @Test
+    public void testAsyncExceptionCanBeHandledDuringRestoring() throws Exception {
```

Comment:
```suggestion
    public void testHandleAsyncExceptionDuringRestoring() throws Exception {
```

Reply:
This is just my personal preference, and the current name is also acceptable.

Reply:
LGTM